src/main/resources/index.html

```diff
@@ -269,8 +275,12 @@
             update();
         });
 
+        const debouncedConditionalUpdate = debounce(conditionalUpdate, 250);
```

Comment:
Use a debounce wait time of 300 rather than 250 for consistency with other debounces.